docs/src/pages/components/button/react-auto-prop-table-button.mdx

```diff
@@ -0,0 +1,135 @@
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| alignContent | ResponsiveStyle&#60;AlignContent&#62; | / | Sets the distribution of space between and around content items |
+| alignItems | ResponsiveStyle&#60;AlignItems&#62; | / | Controls where the Flex/Grid items sit on the cross axis. |
+| area | ResponsiveStyle&#60;GridArea&#62; | / |  |
+| basis | ResponsiveStyle&#60;FlexBasis&#60;0 &#124; (string & \{\})&#62;&#62; | / | Default size of element before remaining space is distributed |
+| column | ResponsiveStyle&#60;GridColumn&#62; | / |  |
+| columnEnd | ResponsiveStyle&#60;GridColumnEnd&#62; | / |  |
+| columnGap | ResponsiveStyle&#60;StyleToken&#60;GridColumnGap&#60;0 &#124; (string & \{\})&#62;&#62;&#62; | / | Spacing between Flex/Grid child columns |
+| columnSpan | ResponsiveStyle&#60;GridSpanType&#62; | / |  |
+| columnStart | ResponsiveStyle&#60;GridColumnStart&#62; | / |  |
+| direction | ResponsiveStyle&#60;FlexDirection&#62; | / | Sets how flex items are placed in the flex container defining the main axis and the direction (normal or reversed). (maps to flex-direction CSS property) |
+| flex | ResponsiveStyle&#60;Flex&#60;0 &#124; (string & \{\})&#62;&#62; | / | Shorthand for flex grow / shrink / basis |
+| gap | ResponsiveStyle&#60;StyleToken&#60;Gap&#60;0 &#124; (string & \{\})&#62;&#62;&#62; | / | Spacing between child components. Shorthand for rowGap and columnGap. |
+| grow | ResponsiveStyle&#60;FlexGrow&#62; | / | Ability for flex item to grow |
+| justifyContent | ResponsiveStyle&#60;JustifyContent&#62; | / | Controls where the Flex/Grid items sit on the main axis. |
+| order | ResponsiveStyle&#60;Order&#62; | / | Controls order flex items appear |
+| row | ResponsiveStyle&#60;GridRow&#62; | / |  |
+| rowEnd | ResponsiveStyle&#60;GridRowEnd&#62; | / |  |
+| rowGap | ResponsiveStyle&#60;StyleToken&#60;RowGap&#60;0 &#124; (string & \{\})&#62;&#62;&#62; | / | Spacing between Flex/Grid child rows |
+| rowSpan | ResponsiveStyle&#60;GridSpanType&#62; | / |  |
+| rowStart | ResponsiveStyle&#60;GridRowStart&#62; | / |  |
+| shrink | ResponsiveStyle&#60;FlexShrink&#62; | / | Ability for flex item to shrink |
+| wrap | ResponsiveStyle&#60;FlexWrap&#62; | / | The flexWrap property is set on containers and it controls what happens when children overflow the size of the container along the main axis. By default, children are forced into a single line (which can shrink elements). If wrapping is allowed, items are wrapped into multiple lines along the main axis if needed. (maps to flex-wrap CSS property) |
+
+      `}
+    />
+
+  </ExpanderItem>
+
+<ExpanderItem title="Styling" value="Styling">
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| alignSelf | ResponsiveStyle&#60;StyleToken&#60;AlignSelf&#62;&#62; | / |  |
```

Comment:
Since these props will be the same on every component, can we generate this Styling table into a shared StylingProps.mdx file and import it for each component?

Reply:
Good catch. I just noticed that they are identical.

If it's impossible that they will be different in the future, I'll use a shared file instead